index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE HTML>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" media="print" />
+  <link rel="stylesheet" href="style.css">
+  <title>Technical Documentation Page - FreeCodeCamp</title>
+</head>
+
+<body>
+
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <nav id="navbar" id="Our Topic">
+    <header>Our Topic</header>
+    <a href="#Topic_1" class="nav-link">Topic 1</a>
+    <a href="#Topic_2" class="nav-link">Topic 2</a>
+    <a href="#Topic_3" class="nav-link">Topic 3</a>
+    <a href="#Topic_4" class="nav-link">Topic 4</a>
+    <a href="#Topic_5" class="nav-link">Topic 5</a>
```

Comment:
Kindly use a better reason for this change than "this will make it more professional" and try to be more specific. Example:
> You have a list of elements here and such lists should be used along with ul/li tags.